test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/SchemaCompare/SchemaCompareTestUtils.cs

```diff
@@ -14,20 +14,29 @@
 using System;
 using System.IO;
 using static Microsoft.SqlTools.ServiceLayer.IntegrationTests.Utility.LiveConnectionHelper;
+using System.Collections.Generic;
 
 namespace Microsoft.SqlTools.ServiceLayer.IntegrationTests.SchemaCompare
 {
     internal static class SchemaCompareTestUtils
     {
-        internal static void VerifyAndCleanup(string filePath)
+        internal static void VerifyAndCleanup(string path)
         {
             // Verify it was created
-            Assert.True(File.Exists(filePath), $"File {filePath} was expected to exist but did not"); 
+            Assert.True(File.Exists(path) || Directory.Exists(path), $"File or directory {path} was expected to exist but did not");
 
-            // Remove the file
-            if (File.Exists(filePath))
+            FileAttributes attr = File.GetAttributes(path);
+
+            // If path is a directory, delete it and all subdirs/subfiles
+            if ((attr & FileAttributes.Directory) == FileAttributes.Directory)
+            {
+                new DirectoryInfo(path).Delete(true);
+            }
+
+            // If path is a file, delete it
+            else
```

Comment:
Would prefer not to split up if/else like this. Putting the comments within the blocks is typically how I approach that.